### ISSUE TITLE
A11y: Lack of support for 'up' and 'down' arrow keys

### ISSUE THREAD

**Issue by namansancheti** [ccsCoder/image-accordion#1]
As per the `Keyboard Support` section in the below page, the `Up` and `Down` arrow keys should be supported for an accordion:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html#kbd_label

However, this support is missing.

**Comment by ccsCoder** [comment]
Would you like to work on this?
If yes, it's all yours.

**Comment by namansancheti** [comment]
Sure - will raise PR.
